pkgs/stdenv/generic/setup.sh

```diff
@@ -1021,6 +1021,21 @@ configurePhase() {
             echo "fixing libtool script $i"
             fixLibtool "$i"
         done
+
+        # replace `/usr/bin/file` with `file` in any `configure`
+        # scripts with vendored libtool code.  Preserve mtimes to
+        # prevent some packages (e.g. libidn2) from spontaneously
+        # autoreconf'ing themselves
+        CONFIGURE_MTIME_REFERENCE=$(mktemp configure.mtime.reference.XXX)
```

Comment:
We handle quoting in a very lax way in nixpkgs, but I think at least in places like `setup.sh` we should do it properly.

Reply:
You mean if $TMPDIR contains spaces, I presume?  I guess that is possible.

I am generally pretty paranoid about shell quoting issues which is why I wrote the `find` invocation the way I did.

Reply:
https://github.com/NixOS/nixpkgs/pull/168413/commits/43a087cacce9b4fd9e4bd965441320f06b9b35a6